src/test/resources/application.properties

```diff
@@ -1 +1 @@
-injecttestdata=true
\ No newline at end of file
+appconfig.injecttestdata=true
```

Comment:
remove this line
This configuration should not be persisted in the repository

Reply:
For the 3rd time. thats the test config -.-

Reply:
I see